<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>723</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>640</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>65535</width>
     <height>65535</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="centralLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadVolume"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>185</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>524287</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabController">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabVolumeProperty">
        <attribute name="title">
         <string>Volume Property</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>378</width>
              <height>550</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="title">
                <string>Histogram</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QCustomPlot" name="histogramChart" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>150</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox">
               <property name="title">
                <string>Opacity</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="ctkVTKScalarsToColorsWidget" name="opacityTransferFunc"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_2">
               <property name="title">
                <string>Color</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="ctkVTKScalarsToColorsWidget" name="colorTransferFunc"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabLightProperty">
        <attribute name="title">
         <string>Light</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>378</width>
              <height>550</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_4">
               <property name="title">
                <string>Environment Light</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>background</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>u offset</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="ctkColorPickerButton" name="ColorPickerButton_EnvBackground">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>v offset</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_EnvLightVOffset">
                  <property name="singleStep">
                   <double>0.010000000000000</double>
                  </property>
                  <property name="minimum">
                   <double>-0.500000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>0.500000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="ctkSliderWidget" name="SliderWidget_EnvLightUOffset">
                  <property name="singleStep">
                   <double>0.010000000000000</double>
                  </property>
                  <property name="minimum">
                   <double>-0.500000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>0.500000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="ctkPathLineEdit" name="PathLineEdit_EnvMap"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>EnvMap</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionLoadVolume">
   <property name="icon">
    <iconset resource="../resources/images.qrc">
     <normaloff>:/icons/folder-horizontal-open.png</normaloff>:/icons/folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>LoadVolume</string>
   </property>
   <property name="toolTip">
    <string>load volume data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ctkVTKScalarsToColorsWidget</class>
   <extends>QWidget</extends>
   <header>ctkVTKScalarsToColorsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>gui/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
